common/stats/stats.go

```diff
@@ -521,3 +522,11 @@ func StartUptimeReporting(stat StatsReceiver, statName string) {
 		}
 	}
 }
+
+func ReportServerRestart(stat StatsReceiver, statName string) {
+	stat.Gauge(statName).Update(int64(1))
+	go func(stat StatsReceiver, statName string) {
+		time.Sleep(1 *time.Minute)
```

Comment:
Would it be safer to make this two minutes? Also this should be configurable since 1 minute doesn't mean much in the OSS world.